assets/node.yaml

```diff
@@ -9,6 +9,10 @@ spec:
   selector:
     matchLabels:
       app: aws-ebs-csi-driver-node
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```

Comment:
I dont' get it, aren't these the default values? IIUC the [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1933184), this should be 10% instead.